src/js.rs

```diff
@@ -612,6 +612,82 @@ extern {
     pub fn value_of(this: &Object) -> Object;
 }
 
+// Set
+#[wasm_bindgen]
+extern {
+    pub type Set;
+
+    /// The size accessor property returns the number of elements in a Set object.
+    /// 
+    /// https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Set/size
+    
+    #[wasm_bindgen(method, getter, structural)]
+    pub fn size(this: &Set) -> Number;
```

Comment:
Can you order them alphabetically as mentioned in the Contribution guide?

Reply:
I changed the order accordingly. Why does the ci fail? Same thing happened in my other PR but it worked before.

Reply:
I restarted the build for the job that failed. I've seen it failing for others, for no particular reason. It looks like sometimes the build sometimes doesn't properly kickoff.